.github/workflows/openmdao_test_workflow.yml

```diff
@@ -158,6 +163,13 @@ jobs:
             make strict;
           fi
 
+          echo "=============================================================";
+          echo "Submit coverage";
+          echo "=============================================================";
+          pip install git+https://github.com/swryan/coveralls-python;
```

Comment:
Just checking here that the pip install from your personal branch is intentional.

Reply:
Yes..   I have a modified version of the `coveralls-python` package that adds the `--base-dir` argument so we can run the installed version of `OpenMDAO` versus the dev version, and filter out the `site-packages` part of the path (for both linux and windows)...    I will submit for consideration on the main repo after I figure out a couple of tests...